db/migrate/20151221210743_publish_existing_sets.rb

```diff
@@ -0,0 +1,8 @@
+class PublishExistingSets < ActiveRecord::Migration
+  def self.up
+    SourceSet.update_all(published: true)
+  end
+
+  def self.down
```

Comment:
Probably not.  I was just following an example.  I'll try removing it and see what happens.